### ISSUE TITLE
HaritiInstall doesn't work correctly when current directory isn't under the git control

### ISSUE THREAD

**Issue by 0xBADDCAFE** [kamichidu/vim-hariti#12]
The directory is not under the git control (or hasn't remote git repository), `HaritiInstall` and `HaritiUpdate` don't work collectly.

Is not under the git control:

```
hariti: Start 54 bundles...
hariti: nyancat-vim - error - fatal: Not a git repository (or any of the parent directories): .git^@^@
exit status 128
hariti: nyancat-vim - finish
hariti: vim-operator-user - error - fatal: Not a git repository (or any of the parent directories): .g
it^@^@exit status 128
hariti: vim-operator-user - finish
…
```

Has not remote repository:

```
 HaritiInstall
hariti: Start 54 bundles...
hariti: webapi-vim - error - fatal: No remote repository specified.  Please, specify either a URL or a
^@remote name from which new revisions should be fetched.^@^@exit status 1
hariti: webapi-vim - finish
hariti: sonictemplate-vim - error - fatal: No remote repository specified.  Please, specify either a U
RL or a^@remote name from which new revisions should be fetched.^@^@exit status 1
hariti: sonictemplate-vim - finish
…
```

**Comment by kamichidu** [comment]
I'll check later

**Comment by kamichidu** [comment]
fixed 53e03293db237ffe73fc1d52d3d6ecda07cd8918

HaritiInstall/Update spawn git process with current directory, not each git local repository.
